src/SymmetricPositiveDefinite.jl

```diff
@@ -444,3 +444,25 @@ function check_tangent_vector(M::SymmetricPositiveDefinite{N},x,v; kwargs...) wh
     return nothing
 end
 check_tangent_vector(M::MetricManifold{SymmetricPositiveDefinite{N},LogCholeskyMetric},x,v; kwargs...) where N = check_tangent_vector(M.manifold,x,v;kwargs...)
+
+"""
+    mean(M::SymmetricPositiveDefinite, x::AbstractVector[, w::AbstractWeights]; kwargs...)
+
+Compute the Riemannian
+[`mean`](@ref mean(::Manifold, ::AbstractVector, ::AbstractVector, ::GeodesicInterpolation))
+of `x` using [`GeodesicInterpolation`](@ref).
+"""
+mean(::SymmetricPositiveDefinite, args...)
+
+mean!(M::SymmetricPositiveDefinite, y, x::AbstractVector, w::AbstractVector; kwargs...) =
+    mean!(M, y, x, w, GeodesicInterpolation(); kwargs...)
+
+function mean!(
```

Comment:
This should be done in Metric.jl for each default metric (like here the `LinearAffine`) to fall back to the mean with the base manifold.